docs/README.md

```diff
@@ -119,6 +122,55 @@ Note:
 
 ---
 
+## Developing a LabVIEW measurement client
```

Comment:
I would call this "Generating a LabVIEW measurement client" instead since there isn't really any development involved. The generated code should work without modification.